lib/endpoint/tags/table.js

```diff
@@ -126,6 +126,47 @@ function rowToParameter (i, el) {
     params.type = 'objects[]'
   }
 
+  if (params.name === 'content') {
+    if (/reaction type/.test(params.description) === true) {
+      params.enum = ['+1',
+        '-1',
+        'laugh',
+        'confused',
+        'heart',
+        'hooray']
+    }
+  }
+
+  if (params.name === 'type') {
+    if (/Normally this is a `commit`/.test(params.description) === true) {
+      params.enum = ['commit',
+        'tree',
+        'blob']
+    }
+  }
+
+  if (params.name === 'state') {
+    if (/active/.test(params.description) === true) {
+      params.enum = ['active']
+    }
+  }
+
+  if (params.name === 'default_repository_permission') {
+    if (/organization repositories:/.test(params.description) === true) {
+      params.enum = ['read',
+        'write',
+        'admin',
+        'none']
+    }
+  }
+
+  if (params.name === 'privacy') {
+    if (params.default === 'secret') {
```

Comment:
I think here the conditions are simple enough to combine them into

```
params.name === 'privacy' || params.default === 'secret'
```

Reply:
I think it should be `params.name === 'privacy' && params.default === 'secret'`

Reply:
yes, you are right :)

Reply:
let me update the branch.